lib/api/index.js

```diff
@@ -61,7 +61,6 @@ module.exports = class BoolJSBootstrapper {
     setFolder (key, folder) {
         if (!this.folders.has(key)) {
             this.folders.set(key, folder);
```

Comment:
whats the type of this.folders, and why it has to do double reassignment?

Reply:
It's [FolderList](https://github.com/BoolJS/booljs/blob/master/lib/api/folder/list.js)

The double reassignment is so it's able to if the path is not existing in the default list, first adds a setter, then, sets the folder. Now, I see the issue here.